tests/foreman/api/test_hostgroup.py

```diff
@@ -649,9 +631,7 @@ def test_positive_create_with_group_parameters(self, module_org, module_target_s
 
         :customerscenario: true
 
-        :CaseLevel: Integration
-
-        :BZ: 1710853
+                :BZ: 1710853
```

Comment:
Interestingly, this field was also extra-indented (like the others).

Reply:
yeah that was really painful